wp-graphql.php

```diff
@@ -216,7 +216,7 @@ private function includes() {
 			 * The codeception tests are an example of an environment where adding the autoloader again causes issues
 			 * so this is set to false for tests.
 			 */
-			if ( defined( 'WPGRAPHQL_AUTOLOAD' ) && true === WPGRAPHQL_AUTOLOAD ) {
+			if ( ! class_exists('GraphQL\GraphQL') && defined( 'WPGRAPHQL_AUTOLOAD' ) && true === WPGRAPHQL_AUTOLOAD ) {
```

Comment:
🤔 

So, you're installing GraphQL-PHP as a composer dependency separate from WPGraphQL?

WPGraphQL can only really guarantee that it will work with one specific version of GraphQL PHP, so it makes me a bit nervous to allow folks to install them separately as that feels like it will increase our support burden. 

Happy to learn more about your use case to see if it makes sense to support this situation. 

Right now, WPGraphQL is (somewhat) tightly coupled with a specific version of GraphQL-PHP, and as we move forward we can only really promise to work with the specific version we test against.

Reply:
I'm letting composer handle all the dependencies in my project. The reason is to avoid the same libs being loaded by different plugins and to keep the compiled code size to a minimum (I'm running WordPress in a serverless environment so this is important to keep cold boot times to a minimum).

As such this is really just a convenience. It could also prevent issues if wp-graphql is included alongside another plugin that uses the graphql dependency.

Reply:
I'm just nervous that I'm going to have to bear the weight of support tickets for folks bundling their own version of graphql-php and wondering why a feature in WPGraphQL isn't working. 

I'll need to think about this. Simple change, but could potentially open up support requests that I waste more time on than I should 🤔

Reply:
Ok, sounds sensible - with the `WPGRAPHQL_AUTOLOAD` flag I can make it work in my setup. It might be worth leaving the PR open in case others run into a similar issue inadvertently. Thanks for all your work!